interpreter/llvm/src/include/llvm/Support/Signals.h

```diff
@@ -15,6 +15,7 @@
 #define LLVM_SUPPORT_SIGNALS_H
 
 #include <string>
```

Comment:
Can we probably backport https://github.com/llvm/llvm-project/commit/ff1681ddb303223973653f7f5f3f3435b48a1983 which is essentially the same but alphabetically sorted and clear that will not need to be put in the list of our patches to maintain.

Reply:
I think we should still track it in our set of patches, but we will come back to this in today's team meeting